sections/adguard-specific/annoyances.txt

```diff
@@ -46,6 +46,7 @@ myonlineradio.hu#$#.js-appAdv { display: none !important; }
 myonlineradio.hu#$#.cover-slider-cont { display: block !important; }
 noizz.hu#$#.leavingPopupLayer { display: none !important; }
 noizz.hu#$#body { overflow: auto !important; }
+bme.hu#?#style:contains(eu-cookie-withdraw-banner)
```

Comment:
Do we really need this rule? Why not just `/eu_cookie_compliance.min.js$script,domain=bme.hu`?